### PR TITLE
MSS-1344: Fix jackson vulnerabilites and bump dependencies

### PR DESCRIPTION
**Pull request by frankie297**
Force jackson-databind and jackson-datatype versions to fix some vulnerability issues:
https://app.snyk.io/vuln/SNYK-JAVA-COMFASTERXMLJACKSONDATATYPE-173759
https://app.snyk.io/vuln/SNYK-JAVA-COMFASTERXMLJACKSONCORE-174736

NB: remove after upgrading play app to 2.7 (if we should ever do this)

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -25,14 +25,16 @@ scalacOptions in ThisBuild ++= compilerOptions
 val minJacksonVersion: String = "2.8.9"
 val minJacksonLibs = Seq(
   "com.fasterxml.jackson.core" % "jackson-core" % minJacksonVersion,
-  "com.fasterxml.jackson.core" % "jackson-databind" % minJacksonVersion,
   "com.fasterxml.jackson.core" % "jackson-annotations" % minJacksonVersion,
   "com.fasterxml.jackson.dataformat" % "jackson-dataformat-cbor" % minJacksonVersion
 )
 
-val playJsonVersion = "2.6.9"
+val jacksonDatabind = "com.fasterxml.jackson.core" % "jackson-databind" % "2.9.10.1"
```
Comment (alexduf) [inline build.sbt]: Why not changing the `minJacksonVersion` ?

Comment (frankie297) [reply build.sbt]: I have now, thanks for suggesting this 👍

Comment (frankie297) [reply build.sbt]: I have also just bumped up to the latest version as there was a new vulnerability discovered a few days ago (with version 2.9.10.1)

**Comment by frankie297** [comment]
> Make sure to deploy this to CODE and push a notification before merging 😄 👍

Done ✅and Done ✅

**Pull request merged**